.flake8

```diff
@@ -0,0 +1,12 @@
+[flake8]
+exclude =
+    .git,
+    .pytest_cache,
+    __pycache__,
+    docs
+ignore =
+    # line too long: ignore for now
+    E501,
+    # binary op on new line: combo of pathlib and black causes this
+    W503
+max-line-length = 88
```

Comment:
Why include a max-line-length if we're ignoring? (Also curious about where 88 came from, there are so many ideas around what this should be!)

Reply:
This line length matches black's default line length, so I just stuck with that.  The reason I added the ignore for now is that there are some comments that aren't easy to break up into shorter lines (some URLs, if I recall).  Black will mostly output code that adheres to the rules, but there were some exceptions that need to be fixed manually.

There aren't too many of these, so that's a good call-out.  I can swap these over to use `# noqa: E501` and remove this ignore.